BoxContentSDK/BoxContentSDK/Requests/BOXItemSetCollectionsRequest.m

```diff
@@ -50,53 +78,122 @@ - (instancetype)initFolderSetCollectionsRequestForFolderWithID:(NSString *)folde
                        resource:BOXAPIResourceFolders];
 }
 
+- (instancetype)initFolderSetCollectionsRequestForFolderWithID:(NSString *)folderID
+                                                 collectionIDs:(NSArray *)collectionIDs
+                                                   associateId:(NSString *)associateId
+{
+    return [self initWithItemID:folderID
```

Comment:
minor: have the above method use this method

- (instancetype)initFolderSetCollectionsRequestForFolderWithID:collectionIDs: